extra/foreman-debug-proxy

```diff
@@ -21,15 +25,18 @@ add_files /etc/xinetd.d/tftp
 [ -d "/var/lib/tftpboot" ] && add_cmd "find /var/lib/tftpboot -exec ls -ld {} +" "tftpboot_tree"
 
 # ISC bind
-add_files /etc/named.conf /var/log/named.log
-add_files /var/named/* /var/named/{data,dynamic,slaves}/*
+add_files /etc/named.conf
+add_files /var/log/named.log
+for SUBDIR in /var/named /var/named/{data,dynamic,slaves}; do
+  add_files $SUBDIR/
+done
 
 # Puppet master and CA
 add_files /var/lib/puppet/ssl/certs/$(hostname -f).pem /var/lib/puppet/ssl/certs/ca.pem
-add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/* /var/log/puppet/*.log*
+add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/*
```

Comment:
Can't upstream smart-proxy run with older puppet as well?

Anyway, I suggest filing new issue for this. Here in the PR I just reflect the planned change made to foreman-debug. My change does not bring a regression (I hope) and is independent on the puppet version / puppetlabs dir.